Rollbar/RollbarClient.cs

```diff
@@ -37,14 +37,38 @@ private static Guid ParseResponse(string stringResult)
 
         private string SendPost<T>(string url, T payload)
         {
-            var webClient = new WebClient();
+            var webClient = this.BuildWebClient();
             return webClient.UploadString(new Uri($"{Config.EndPoint}{url}"), JsonConvert.SerializeObject(payload));
         }
 
         private async Task<string> SendPostAsync<T>(string url, T payload)
         {
-            var webClient = new WebClient(); 
+            var webClient = this.BuildWebClient();
             return await webClient.UploadStringTaskAsync(new Uri($"{Config.EndPoint}{url}"), JsonConvert.SerializeObject(payload));
         }
+
+        private WebClient BuildWebClient()
```

Comment:
This was the approach I had in my head after reading the docs, so looks good to me.